lerna.json

```diff
@@ -2,7 +2,8 @@
   "packages": [
     "packages/cactus-*",
     "examples/cactus-*",
-    "extensions/cactus-*"
+    "extensions/cactus-*",
+    "examples/carbon-accounting/pkg/cactus-*"
```

Comment:
@jagpreetsinghsasan This line can be removed due to other changes that were made on the main branch since.